tests/test_torch_compile.py

```diff
@@ -349,10 +351,10 @@ def test_causal_lm_multiple_lora_adapter_compile(self, tokenizer, data):
             output_base = model(sample)
 
         config = LoraConfig(task_type=TaskType.CAUSAL_LM, init_lora_weights=False)
-        model = get_peft_model(model, config).eval()
-        model = self.compile(model, {})
+        model = get_peft_model(model, config)
         model.add_adapter("other", config)
         model = self.compile(model, {})
+        model.eval()
```

Comment:
Is it a requirement to only call compile once?

Reply:
From my testing, `torch.compile` needs to be called after loading all adapters. I extended the docs to say so in my [last commit](https://github.com/huggingface/peft/pull/2332/commits/d9cb9f9aa70b63f397e67407c81fac58bc8c71d7).